crates/fluvio-stream-model/src/store/dual_store.rs

```diff
@@ -282,7 +289,6 @@ where
         drop(write_guard);
 
         self.event_publisher.store_change(epoch);
-        self.event_publisher.notify();
```

Comment:
reasonable for this removal?

Reply:
Every time store_change is called, notify() is also called.  I thought it would be simpler and safer to include automatically notify listeners whenever store_change is called instead of requiring users of the interface to remember to notify after storing_changes.

I also removed the increment function since it isn't used anywhere.

Reply:
Make sense.  Nice change